mod/dfrn_request.php

```diff
@@ -611,7 +611,7 @@ function dfrn_request_content(App $a)
 		} elseif (x($_GET, 'address') && ($_GET['address'] != "")) {
 			$myaddr = $_GET['address'];
 		} elseif (local_user()) {
-			if (strlen($a->path)) {
+			if (strlen($a->urlpath)) {
```

Comment:
What is the meaning of this change?

Reply:
`path` was ambiguous, it could also mean the filesystem path. This change makes it clearer which path it is about, the URL base path after the domain name.